### ISSUE TITLE
Order Rule incorrectly warns on no empty line for mutli line imports

### ISSUE THREAD

**Issue by mjrussell** [benmosher/eslint-plugin-import#346]
If you have the following in your .eslintrc:

`import/order: 2`

single line imports work great, but if you have mutliline imports due to max width, order incorrectly warns.

For example:

```js
import { 
  foo, bar,
  buzz,
} from './foo';
import cheese from './cheese';
```

You get an invalid warning. It should only error when there is an unnecessary newline 

There are also invalid warnings on comments such as:
```js
import foo from './foo';
// Comment about import below
import bar from './bar';
```

I don't believe either of these should trigger an error

**Comment by mjrussell** [comment]
Ah I just saw #339. @singles does your PR also allow comments on their own line between import groups?

Looks like multi line imports are fixed in that PR

**Comment by singles** [comment]
@mjrussell This works fine (`newlines-between` set to `never`)

```js
var path = require('path');
//some comment here
require('something-else');
var _ = require('lodash');
```

Code iterates over lines as a strings, trims those and if after trimming it has `length === 0`, then it's considered empty. In case above comment will not be empty and therefore error in this case will not be reported. 

Yes, multiline imports were fixed in mentioned PR, sorry missing that earlier :)

**Comment by mjrussell** [comment]
Cool thanks will test and validate when the new version is released

**Comment by singles** [comment]
@mjrussell Did new version `1.8.1` solved your problem? If yes, we could close this ticket.

**Comment by mjrussell** [comment]
@singles I'll check and report back. Thanks for the note about the new release

**Comment by mjrussell** [comment]
It does indeed solve the issue, closing. Thanks!
